python/cuspatial/cuspatial/core/geoseries.py

```diff
@@ -536,3 +563,150 @@ def to_arrow(self):
                 arrow_polygons,
             ],
         )
+
+    def contains(self, other, align=True):
+        """Compute from a set of points and a set of polygons which points fall
```

Comment:
set --> series? Should this be "GeoSeries"?

```suggestion
        """Compute from a series of points and a series of polygons which points fall
```

Reply:
Yes, done.